style.css

```diff
@@ -0,0 +1,1158 @@
+  background-image: url("assets/contact_form_shapes.png"), url("assets/Contact_me.png");
+  background-size: auto 100%;
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
+  align-self: center;
+}
+
+.contact-info {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-info h2 {
+  font-size: 40px;
+  text-align: center;
+  color: white;
+  margin-top: 160px;
+  margin-bottom: 20px;
+}
+
+.contact-info p {
+  font-size: 20px;
+  text-align: center;
+  margin-bottom: 38px;
+  padding: 0 70px;
+  color: white;
+}
+
+.contact-input {
+  width: 87.2%;
+  height: 48px;
+  margin-bottom: 20px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 8px;
+  font-size: 17px;
+  border: none;
+  color: #00050e;
+  text-indent: 10px;
+}
+
+.contact-textarea {
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  border: none;
+  color: #00050e;
+  text-indent: 10px;
+  padding-top: 10px;
+  margin-bottom: 10px;
+  width: 87.2%;
+  border-radius: 8px;
+  text-decoration: none;
+}
+
+.contact-textarea:focus {
+  border: 1px solid #6070ff;
+}
+
+.contact-button:hover {
+  background-color: #fff;
+  border: 1px solid white;
+  color: white;
+  font-size: 17px;
+}
+
+.input-container {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  margin-left: 24px;
+  margin-right: 24px;
+  align-self: flex-start;
+}
+
+.btn-get-it-touch {
+  text-decoration: none;
+  color: #6070ff;
+  border: 1px solid #396df2;
+  padding: 7px;
+  line-height: 30px;
+  font-size: 17px;
+  border-radius: 8px;
+  margin-bottom: 20px;
+  background-color: #fff;
+}
+
+.btn-get-it-touch:disabled {
+  font-weight: 300;
+  font-size: 17px;
+  font-family: 'Poppins', sans-serif;
+  background: #ebebff;
+  color: #808080;
+  border: 2px solid #808080;
+  border-radius: 4px;
+  padding: 7px;
+}
+
+.btn-get-it-touch:hover {
+  border: 1px solid #396df2;
+  padding: 7px;
+  border-radius: 4px;
+  font-size: 17px;
+  font-weight: 300;
+  background: #396df2;
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+}
+
+.btn-get-it-touch:active {
+  padding: 7px;
+  border-radius: 4px;
+  font-size: 17px;
+  font-weight: 300;
+  background: #000080;
+  color: #ebebff;
+  font-family: 'Poppins', sans-serif;
+}
+
+.btn-get {
+  padding-top: 30px;
+  height: 90px;
+  margin-left: 7px;
+}
+
+.get-it {
+  align-self: flex-start;
+}
+
+.no-padding {
+  padding: 1px;
+}
+
+.img-style-desktop-popup {
+  display: none;
+}
+
+.popup {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(76, 78, 82);
+  background-color: rgb(76, 78, 82, 0.2);
+}
+
+.popup-content {
+  background-color: #fefefe;
+  margin: 15% auto;
+  padding: 20px;
+  border: solid rgb(219, 214, 214) 1px;
+  border-radius: 15px;
+  width: 80%;
+}
```

Comment:
- [ ] Kindly make sure the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) like for the **background color and opacity**, **spacing at the top**, **color on some text content**,...

- [ **OPTIONAL** ] I think it would be better if you remove the `scrolling` effect of the **body content** when you are watching the popup window so the user can not interact with the `body content` and also better if you only leave `the overflow-y` on the popup and remove the one at the bottom(overflow-x) as it not needed and to make the popup looks smart i.e only use `the overflow-y` for scrolling the popup. I know you can do it :muscle: :muscle: